pt-BR/action_controller_overview.md

```diff
@@ -1063,21 +1063,21 @@ Matching URLs will be marked as '[FILTERED]'.
+A excessão padrão do Rails apresenta a mensagem "500 Server Error" para todas as excessões. Se a requisição for feita localmente, um belo TRACEBACK e outras informações serão mostradas assim você pode verificar o que deu errado e tratar o problema. Se a requisição for remota o Rails apenas apresentará a mensagem "500 Server Error" para o usuário, ou um "404 Not Found" se houver um erro na rota ou o registro não puder ser encontrado. As vezes você pode querer customizar como esses erros são encontrados e como são apresentados ao usuário. Há diversos níveis de tratamento de excessões disponiveis em uma aplicação Rails:
 
-### The Default 500 and 404 Templates
+### Os templates 404 e 500 padrão
 
-By default a production application will render either a 404 or a 500 error message, in the development environment all unhandled exceptions are raised. These messages are contained in static HTML files in the public folder, in `404.html` and `500.html` respectively. You can customize these files to add some extra information and style, but remember that they are static HTML; i.e. you can't use ERB, SCSS, CoffeeScript, or layouts for them.
+Por padrão uma aplicação em produção irá renderizar uma mensagem em um template de erro 404 ou 500, no ambiente de produção todas as mensagens de erro são disparadas. Essas mensagens são armazenadas em templates estáticos de html na pasta *public*, em `404.html` e `500.html` respectivamente. Você pode customizar essas páginas e adcionar algumas estilizações, mas lembre-se elas são html estático; i.e. você não pode usar ERB, SCSS, CoffeeScript, ou layouts para elas.
 
 ### `rescue_from`
 
-If you want to do something a bit more elaborate when catching errors, you can use `rescue_from`, which handles exceptions of a certain type (or multiple types) in an entire controller and its subclasses.
+Se você quiser fazer algo mais elaborado quando estiver lidando com erros, você pode usar `rescue_from`, qu trata as excessões de um certo tipo (ou de vários tipos) em um controller inteiro e nas subclasses.
 
-When an exception occurs which is caught by a `rescue_from` directive, the exception object is passed to the handler. The handler can be a method or a `Proc` object passed to the `:with` option. You can also use a block directly instead of an explicit `Proc` object.
+Quando uma excessão acontece e é pega por uma diretiva `rescue_from`, o objeto da excessão é passado ao handler. O handler pode ser um método ou um objeto `Proc` passado com a opção `:with`. Você também pode usar um bloco diretamente ao invés de um objetco `Proc`.
```

Comment:
Aqui é a mesma coisa do comentário anterior, junto com colocar `handler`em itálico e um pequeno typo que eu achei: 💅 

```suggestion
Quando uma exceção acontece e é pega por uma diretiva `rescue_from`, o objeto da exceção é passado ao *handler*. O *handler* pode ser um método ou um objeto `Proc` passado com a opção `:with`. Você também pode usar um bloco diretamente ao invés de um objeto `Proc`.
```